lib/ghtorrent/ghtorrent.rb

```diff
@@ -1660,19 +1661,46 @@ def ensure_issue_comment(owner, repo, issue_id, comment_id, pull_req_id = nil)
         end
 
         user = ensure_user(retrieved['user']['login'], false, false)
+        reactions = retrieved['reactions']
 
         db[:issue_comments].insert(
             :comment_id => comment_id,
             :issue_id => issue[:id],
             :user_id => unless user.nil? then user[:id] end,
-            :created_at => date(retrieved['created_at'])
+            :created_at => date(retrieved['created_at']),
+            :like => reactions['+1'],
+            :dislike => reactions['-1'],
+            :laugh => reactions['laugh'],
+            :confused => reactions['confused'],
+            :heart => reactions['heart'],
+            :hooray => reactions['hooray']
         )
 
         info "Added issue_comment #{issue_comment_str}"
         db[:issue_comments].first(:issue_id => issue[:id],
                                    :comment_id => comment_id)
       else
         debug "Issue comment #{issue_comment_str} exists"
+        if ! (curcomment[:like] == reactions['+1'] or
```

Comment:
i'm having a lot of trouble testing this code block because of the constraint violations mentioned in https://github.com/gousiosg/github-mirror/issues/52